pom.xml

```diff
@@ -55,7 +55,7 @@
 			we need to compile against version 23. All such classes are in the org.acra.jraf package.
 		-->
         	<android.version>23</android.version>
-		<support.version>23.2.1</support.version>
+		<support.version>23.3.0</support.version>

```

Comment:
As long as we don't actually use any new features (in which case we would have to bump it), everyone is free to use an earlier version. I bumped the version because otherwise users (like me) who don't use this library in their project and don't have it as first-level dependency won't receive the bugfixes.
Google Policy advises to always use the latest version of the library and 23.3.0 is mainly a bugfix release.
In fact Android Studio always shows a warning if I'm not building with the latest version.